lib/solidus_dev_support/templates/extension/README.md

```diff
@@ -42,14 +43,13 @@ Simply add this require statement to your spec_helper:
 require '<%= file_name %>/factories'
 ```
 
-Sandbox app
------------
+## Running the sandbox
 
-To run this extension in a sandboxed Solidus application you can run `bin/sandbox`
-The path for the sandbox app is `./sandbox` and `bin/rails` will forward any Rails command
-to `sandbox/bin/rails`.
+To run this extension in a sandboxed Solidus application, you can run `bin/sandbox`. The path for
+the sandbox app is `./sandbox` and `bin/rails` will forward any Rails commands to
+`sandbox/bin/rails`.
```

Comment:
No an error in the context of this PR, but we should note that it's forwarded for all `rails` commands except generators.